wiki/Terminology in Oppia.md

```diff
@@ -0,0 +1,42 @@
+## Table of Contents
+
+- [Overview of entities](#overview-of-entities)
+- [Key terms](#key-terms)
+
+## Overview of entities
+
+This diagram outlines the various entities in Oppia and how they relate
+to each other:
+
+```mermaid
+flowchart TD
+T("Topic") --> S1("Story-1")
+T --> S2("Story-2")
+S1 --> C1("Chapter-1/Exploration")
+C1 --> c1("Card-1")
+C1 --> c2("card-2")
+S1 --> C2("chapter-2")
+C2 --> c3("Card-3")
+c3 --> co("Content")
+c3 ----> in("Interaction")
+in --> mc("Multiple-choice")
+in --> is("Image Selection")
+in --> fi("Fraction-input")
+in --> oqt("other question types")
+S2 --> C3("Chapter-3")
+S2 --> C4("Chapter-4")
+C3 --> c4("Card-4")
+C4 --> c5("Card-5")
+C4 --> c6("Card-6")
+```
+
+## Key terms
+
+1. **Topic**: A topic is a broad term that refers to the subject content being taught (e.g. Addition/Subtraction). It's appear on Home screen of app.
+2. **Story**: Stories are situations/scenarios that are meant to help users understand the topic. For example, if the topic is addition, then one of the stories could be about where a kid goes to a shop and buys 3 pens and 4 pencils. It's appear when you click any "Topic" in "Lessons" tab.
+3. **Chapter**: A chapter corresponds to different lessons through which we aim to teach the topic to the users. Each chapter has an associated exploration, and this exploration may have several cards.
+4. **Promoted Story**: Promoted Story is mainly the recent Story/Chapter you played. It's shown on "home screen" with heading text "Stories For You".
+5. **Skill**: This is a concrete learning outcome that describes something that a learner should be able to do. It is usually stated in the form “Given X, compute/calculate/draw/etc. Y.” For example: “Given a fraction, identify its numerator.”
+6. **Exploration**: This is a structured learning experience that is part of a story, and provides the learner with an active way to learn new concepts, as well as targeted feedback. It is the core unit of learning in Oppia. The flow/screens appears when any story is started is known as Exploration.
```

Comment:
Isn't a chapter and exploration the same? You can combine the two definitions.

Reply:
Yes, Right. Thanks!
Updated.